src/scripts/site.js

```diff
@@ -1240,29 +1217,29 @@ var AppInfo = {};
                     }
 
                     if (!browser.tv && !browser.xboxOne && !browser.ps4) {
-                        require(["bower_components/emby-webcomponents/nowplayingbar/nowplayingbar"]);
+                        require(["components/nowplayingbar/nowplayingbar"]);
                     }
 
                     if (appHost.supports("remotecontrol")) {
-                        require(["playerSelectionMenu", "bower_components/emby-webcomponents/playback/remotecontrolautoplay"]);
+                        require(["playerSelectionMenu", "components/playback/remotecontrolautoplay"]);
                     }
 
                     if (!(appHost.supports("physicalvolumecontrol") && !browser.touch || browser.edge)) {
```

Comment:
I wonder why we don't show volumecontrol for Edge... is it intentional, or a bug?

Reply:
Digging a little deep, i found that the control event for video html5 player is volumechange. This event is introduced in HTML5 specification, so it should be supported in edge without major problems. I think is fine to remove edge in this condition. Should i remove it?

Reply:
I can only think of some funny edge (pun unintended) cases like Edge running as default browser on XBox maybe?..

Reply:
I can't test this for real and a real search on the web shows nothing regarding such issue. Edge should support html5 video without problems per spec: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video

Reply:
Then I suggest removing it, we can always fix it again later if things break.

Reply:
condition removed.